Sources/Coordination/ParentCoordinator.swift

```diff
@@ -13,7 +13,7 @@ public protocol ParentCoordinator: Coordinator {
 }
 
 extension ParentCoordinator {
-    func openChild(_ childCoordinator: ChildCoordinator) {
+    public func openChild(_ childCoordinator: ChildCoordinator) {
```

Comment:
```suggestion
    /// openChild
    /// Start tracking a new child coordinator.
    /// - Parameter child: The child coordinator to be tracked
    ///
    /// This method should only be called directly if implementing a custom coordinator presentation.
    /// In other cases you should call the relevant openChild method for the relevant presentation style.
    /// For example: `openChildModally` should be used for modal presentation.
    ///
    public func openChild(_ childCoordinator: ChildCoordinator) {
```

Reply:
Appreciated, will add some documentation to the README as well 👍